### DISCUSSION TITLE
Pixi 6.5.2, typescript, webpack - not able to load spine files

### DISCUSSION THREAD

**Discussion by amitmpixentech** [pixijs/pixijs#8534] [General]
Hi,

Hi,

I have used pixi spine for years but recently I have moved to typescript and now my loader loads the spine json file but not the atlas and hence getting error. My package.json has following entries 👍
"dependencies": {
 "@pixi-spine/all-3.8": "^3.0.16",
    "gsap": "^3.10.4",
    "howler": "^2.2.3",
    "pixi.js": "^6.5.0"
},

my tsconfig file looks like :
{
    "compilerOptions": {
        "target": "esnext",
        "module": "commonjs",
        "removeComments": true,
        "sourceMap": true,
        "watch": false,
        "outDir": "./public/js",
        "moduleResolution": "node"
    },
    "filesGlob": [
        "src/**/*.ts"
    ],
    "exclude": [
        "node_modules"
    ]
}

I have tried multiple spine files which are working fine in other projects but not in this typescript project. Any help would be really appreciated as I am totally clueless what is going on.

const app = new Application({
      width: 1920,
      height: 1080,
      resolution: window.devicePixelRatio || 1,
      backgroundColor:0x000000,
      resizeTo: window,
    });
    document.body.appendChild(app.view);

    let loader = app.loader;

    loader.add('win', 'gameassets/1.00/spines/assets/Wins.json')
    loader.onComplete.add(onAssetsLoaded)
    loader.load();

This is how I am loading the spine file.

My doubt is spine loader is not getting registered somehow and hence it is not able to understand that it needs to load the atlas. Please suggest some solution. I am using Pixijs 6.5 with webpack and ES6.

**Comment by SirMaxeN** [comment]
I had same problem with 6.4.2
check here
https://github.com/pixijs/spine/issues/448

Basically set module in compilerOptions to es6 in tsconfig.json ```"module": "es6",```
